src/ClientHandle.cpp

```diff
@@ -2704,7 +2704,20 @@ void cClientHandle::SendResetTitle()
 
 void cClientHandle::SendRespawn(eDimension a_Dimension, bool a_ShouldIgnoreDimensionChecks)
 {
-	m_Protocol->SendRespawn(a_Dimension, a_ShouldIgnoreDimensionChecks);
+	eDimension CurrentDimension = m_Player->GetWorld()->GetDimension();
```

Comment:
Should I use m_LastSentDimension instead? Or should I simply remove `m_LastSentDimension`?